src/ledgerwriter/src/main/java/anthos/samples/bankofanthos/ledgerwriter/LedgerWriterApplication.java

```diff
@@ -72,4 +77,43 @@ public RestTemplate restTemplate() {
     public void destroy() {
         LOGGER.info("LedgerWriter service shutting down");
     }
+
+    /**
+     * Initializes Meter Registry with custom Stackdriver configuration
+     *
+     * @return the StackdriverMeterRegistry with configuration
+     */
+    @Bean
```

Comment:
is it possible to share this Bean across the 3 java services? or do you think it's more trouble than it's worth?